src/universal/modules/welcome/containers/Welcome/Welcome.js

```diff
@@ -62,46 +90,55 @@ export default class WelcomeContainer extends Component {
     const teamId = shortid.generate();
     const teamMemberId = shortid.generate();
     const {user} = getAuth();
-    dispatch(setWelcomeTeam({teamName, teamId, teamMemberId}));
+    dispatch(setWelcomeTeam({teamId, teamMemberId}));
     const createTeamOptions = {
       variables: {
         newTeam: {
           id: teamId,
           name: teamName,
-          leader: {
+          leader: [{
             id: teamMemberId,
             teamId,
             cachedUserId: user.id,
             isActive: true,
             isLead: true,
             isFacilitator: true
-          }
+          }]
         }
       }
     };
     cashay.mutate('createTeam', createTeamOptions);
+    this.nextPage();
   };
 
   onInviteTeamSubmit = data => {
     const {dispatch, welcome: {teamId}} = this.props;
-    const {invitees} = data;
+    const invitees = data.invitees.map(i =>
+      // Remove label field:
+      pick(i, 'email', 'fullName', 'task')
+    );
     const options = {
       variables: {
         teamId,
         invitees
       }
     };
-    cashay.mutate('inviteTeam', options)
+
+    cashay.mutate('inviteTeamMembers', options)
       .then(res => {
         // TODO make sure this resolves after the route changes
         console.log('inviteTeamRes', res);
         if (res.error) {
-          const {failedEmails} = JSON.parse(res.error);
```

Comment:
Error handling isn't working yet. @mattkrick can you fix this pattern before we merge?